aten/src/ATen/native/cuda/ReduceMomentKernel.cu

```diff
@@ -62,6 +62,12 @@ static void mean_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+template <typename scalar_t, typename acc_t=scalar_t, typename out_t=scalar_t>
+void nanmean_kernel_impl(TensorIterator& iter) {
+  float factor = float(iter.num_output_elements()) / iter.numel();
```

Comment:
TODO: What is the use of `factor` here?

Also:
`gpu_reduce_kernel<scalar_t, out_t>(iter, MeanOps<acc_t, float> {factor});` - where is `MeanOps` defined?